content/news.pml

```diff
@@ -39,6 +39,25 @@
 
 <!----------------------------------------------------------->
 
+<div class="news-item">
+  13 July 2021: TU Munich joins seL4 Foundation
+</div>
+<div>
+ <p>
+   <img src="/Foundation/Membership/LOGOS/TU-Munich.svg"
+   style="width: 15%;  padding-left:10px;  float:right"
+	alt="TU Munich logo">
+
+   The seL4 Foundation is pleased to
+   welcome <a href="https://www.tum.de/en">ETH Zurich</a> as
```

Comment:
uhm, I assume it should say `Technical University of Munich` (or 'TU München') here and not `ETH Zurich`?

Reply:
Thanks for that catch, obviously a C&P error that would have been rather embarrassing.